homeassistant/components/lg_thinq/strings.json

```diff
@@ -290,26 +325,26 @@
         "name": "{location} night growth temperature"
       },
       "temperature_state": {
-        "name": "[%key:component::sensor::entity_component::temperature::name%]",
+        "name": "Temperature",
         "state": {
           "high": "High",
           "normal": "Good",
           "low": "Low"
         }
       },
       "temperature_state_for_location": {
-        "name": "[%key:component::lg_thinq::entity::number::target_temperature_for_location::name%]",
+        "name": "{location} temperature",
         "state": {
-          "high": "[%key:component::lg_thinq::entity::sensor::temperature_state::state::high%]",
-          "normal": "[%key:component::lg_thinq::entity::sensor::temperature_state::state::normal%]",
-          "low": "[%key:component::lg_thinq::entity::sensor::temperature_state::state::low%]"
+          "high": "High",
+          "normal": "Good",
+          "low": "Low"
         }
```

Comment:
Please revert the change of the references

Reply:
Reverted.